week-2-practise/level-3/5-indexOf.js

```diff
@@ -6,7 +6,8 @@
 // - return a new array with the replacement value inserted
 
 function replace(arr, valueToReplace, newValue) {
-  return; // complete this statement
+  index = arr.indexOf(valueToReplace);
+  return arr.slice(0, index).concat(newValue, arr.slice(index + 1)); // complete this statement
```

Comment:
Nice that you've kept the index as a separate variable and modified your previous solution for the slices.